go/engine/buffered_identify_ui.go

```diff
@@ -77,12 +78,18 @@ func (b *bufferedIdentifyUI) flush(trackingBroke bool) (err error) {
 		return nil
 	}
 
+	if !b.G().IdentifyUILimiter.Wait(5 * time.Millisecond) {
```

Comment:
Maybe the solution to my concern is no to allow ratelimiting if you track them --- and to only use it if you don't...

Reply:
In other words, @patrickxb, my proposal is to delete these 3 lines...

Reply:
Ok.